app/controllers/webhooks_controller.rb

```diff
@@ -5,17 +5,79 @@
 class WebhooksController < ApplicationController
   # POST /webhooks
   def create
-    if request.env['HTTP_STRIPE_SIGNATURE'].present?
-      handle_stripe_event
-    elsif request.env['HTTP_X_SQUARE_SIGNATURE'].present?
-      handle_square_event
-    end
 
-    json_response({})
+    api_key = "" # Insert API key here
+    api_url = "https://api:#{api_key}@api.mailgun.net/v2/m.sendchinatownlove.com/messages"
+
+    amount = 500
+    merchant = "Joolie's Bakery"
+    # receipt_id = "blerghllejs"
+    # receipt_url = "coolmath.com"
+
+    amount_string = '%.2f' % ((amount.to_f)/100)
+
+    html = "<!DOCTYPE html>" +
```

Comment:
Wait this is miserable. Is it possible to write it has an html file and write a script to convert it to this upon deploy? @stanleyzhengnyc If I wrote a Python script or Perl script or something, can we run it as a part of the build?

Reply:
Matter of fact, we can just write an HTML file and read it for each email. It's not efficient, but we're talking like 3-6 donations a day.

Reply:
not a rails expert we can load it as a HTML view as an ERB template,we should be able to just load and drop a new file there
like here 
https://github.com/sendchinatownlove/ruby/blob/master/app/views/layouts/receipt.html_A.mjs
that being said its just a rail blocker we're not really familiar with; I looked into it and im not 100% sure how to do it 

https://guides.rubyonrails.org/layouts_and_rendering.html

Reply:
@Jeremy-Xue are you familiar more with rails how to get this email html mailer view

Reply:
Yeah totally agree this is sad state of affairs. I don't know how to use a parse-from-html helper that can also inserting our own vars into the html (like seller name, etc). 

The thing is, the smtp/Action Mailer method has all this capability built right in - using an html.erb view file that can take in vars. We just still have those auth issues with the smtp method. The receipts are so cute I thought it might be worth getting it out in this format and then coming back to this later and trying again to move it to smtp when other stuff isn't so pressing, maybe trying to get dev help from mailgun or something. 

@Jeremy-Xue if you know an http api way to do with with a mailer view that would be so awesome!

Reply:
Taking a look at this rn!

Reply:
I've done something similar to this with django's mailer but haven't used a rails mailer or mailgun before :( 

I think I agree with the idea to deploy the current version come back to it later

Reply:
Maybe this is too late but Ruby has a gem for templates: https://github.com/chamnap/liquid-rails. 

I use it when I styled email templates for work. You can also define `drop` files which are basically where you would declare variables that you can use in your liquid template files.